### PR TITLE
Cybereason command addition

### PR DESCRIPTION
**Pull request by shabina-metron**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py [hunk 1]
```diff
@@ -63,6 +63,10 @@
 MALOP_HEADERS = [
     'GUID', 'Link', 'CreationTime', 'Status', 'LastUpdateTime', 'DecisionFailure', 'Suspects', 'AffectedMachine', 'InvolvedHash']
 
+SINGLE_MALOP_HEADERS = [
+    'GUID', 'Link', 'CreationTime', 'Status', 'LastUpdateTime', 'InvolvedHash']
+
+
```
Comment (suraj-metron) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason.py]: Please remove this extra line, if not needed

File: Packs/Cybereason/ReleaseNotes/2_1_17.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+
+##### Cybereason
+
+- Added a new command ***cybereason-process-attack-tree*** to fetch Process attack tree url.
```
Comment (suraj-metron) [inline Packs/Cybereason/ReleaseNotes/2_1_17.md]: Replace "Added a new command ***cybereason-process-attack-tree*** to fetch Process attack tree url." with
"Added a new command ***cybereason-process-attack-tree*** to return Attack tree url for a given Process."

File: Packs/Cybereason/ReleaseNotes/2_1_17.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+- Added a new command ***cybereason-query-malop-management*** to fetch details of single malop.
```
Comment (suraj-metron) [inline Packs/Cybereason/ReleaseNotes/2_1_17.md]: Replace "Added a new command ***cybereason-query-malop-management*** to fetch details of single malop." with
Added a new command ***cybereason-query-malop-management*** to fetch details of a single management Malop.

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.yml [hunk 4]
```diff
@@ -362,6 +362,19 @@ script:
     - contextPath: Cybereason.Malops.Status
       description: Malop managemant status.
       type: string
+  - arguments:
+    - description: malopGuid of the Cybereason Malop.
+      name: malopGuid
+      required: true
+    description: Get Malop Details.
```
Comment (suraj-metron) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason.yml]: Update description to "Get Management Malop details"

File: Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py [hunk 5]
```diff
@@ -283,6 +283,47 @@ def test_query_malops_command(mocker):
...
+    client = Client(
+        base_url="https://test.server.com:8888",
+        verify=False,
+        headers=HEADERS,
+        proxy=True)
+    args = {
+        "processGuid": "HobXaEWU0CZ6S6LC"
+    }
+    url = "https://test.server.com:8888/#/processTree?guid=HobXaEWU0CZ6S6LC&viewedGuids=HobXaEWU0CZ6S6LC&rootType=Process"
+    test_reponse = [
```
Comment (suraj-metron) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py]: Rename the variable name to "expected_response"

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.yml [hunk 6]
```diff
@@ -362,6 +362,19 @@ script:
...
+    name: cybereason-query-malop-management
+  - arguments:
+    - description: processGuid of the Cybereason Malop.
```
Comment (suraj-metron) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason.yml]: replace description with "processGuid of the Cybereason Malop (can accept multiple guids separated by comma)"

**Review by suraj-metron — CHANGES_REQUESTED** [review]

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py (hunk 1 above)
Comment (shabina-metron) [reply Packs/Cybereason/Integrations/Cybereason/Cybereason.py]: Removed

File: Packs/Cybereason/ReleaseNotes/2_1_17.md (hunk 2 above)
Comment (shabina-metron) [reply Packs/Cybereason/ReleaseNotes/2_1_17.md]: Replaced

File: Packs/Cybereason/ReleaseNotes/2_1_17.md (hunk 3 above)
Comment (shabina-metron) [reply Packs/Cybereason/ReleaseNotes/2_1_17.md]: Replaced

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.yml (hunk 4 above)
Comment (shabina-metron) [reply Packs/Cybereason/Integrations/Cybereason/Cybereason.yml]: Updated

File: Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py (hunk 5 above)
Comment (shabina-metron) [reply Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py]: Renamed

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.yml (hunk 6 above)
Comment (shabina-metron) [reply Packs/Cybereason/Integrations/Cybereason/Cybereason.yml]: Replaced

**Review by suraj-metron — APPROVED** [review]

**Review by sourabh-metron — APPROVED** [review]

**Pull request merged**